docs/usage/histogram.rst

```diff
@@ -16,10 +16,15 @@ All storages support a ``weight=`` parameter, and some storages support a ``samp
 
 The summing accumulators (not ``Mean()`` and ``WeightedMean())``) support threaded filling. Pass ``threads=N`` to the fill parameter to fill with ``N`` threads (and using 0 will select the number of virtual cores on your system). This is helpful only if you have a large number of entries compared to your number of bins, as all non-atomic storages will make copies for each thread, and then will recombine after the fill is complete.
 
+Data
+^^^^
+
+The primary value from a histogram is always available as ``.value()``. The variance is available as ``.variances()``, unless you fill an unweighed histogram with weights, which will cause this to be return None, since the variance is no longer computable (use a weighted storage instead if you need the variances). The effective counts are available as ``.counts()``; this
+
```

Comment:
Incomplete sentence?

Please just write the "The counts are available as ...". And then "If the histogram is weighted, ``.counts()`` returns  the effective counts see docstring for details.

Reply:
Sounds good, but we should link or provide the explanation in the docs somewhere (possibly linking to the docstring in the API docs, if double, would be best); docs shouldn't ask users to look at the docstring _in_ Python; a user is not always in a REPL.